### PR TITLE
Fix: compress health diagnostics file when download

### PR DESCRIPTION
**Pull request by Alevsk**
- subnet expects the generated health diagnostics file to be gz compressed to work

Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>

### CODE REVIEW

File: restapi/admin_health_info.go [hunk 1]
```diff
@@ -51,19 +55,60 @@ func startHealthInfo(ctx context.Context, conn WSConn, client MinioAdmin, deadli
...
 		return err
 	}
 
 	// Send Message through websocket connection
-	return conn.writeMessage(websocket.TextMessage, bytes)
+	return conn.writeMessage(websocket.TextMessage, message)
+}
+
+// compress and tar MinIO diagnostics output
+func tarGZ(healthInfo interface{}, version string) ([]byte, error) {
```
Comment (Alevsk) [inline restapi/admin_health_info.go:90]: @klauspost can you check if this is the right way to implement the compression functionality from your library?

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
I'm getting errors trying to uncompress the file, is this expected?
Screen Shot 2022-04-11 at 12 07 54
Screen Shot 2022-04-11 at 12 07 47

**Comment by bexsoft** [comment]
> > I'm getting errors trying to uncompress the file, is this expected?
> > Screen Shot 2022-04-11 at 12 07 54
> > Screen Shot 2022-04-11 at 12 07 47
> 
> Can you please try it on incognito window or chrome browser to see if this is safari related , I cannot replicate it

I'm using Chrome, Also tried with Firefox & Safari both on 5005 (with WS enabled) & 9090, same results are seen

**Comment by dvaldivia** [comment]
same problem on `safari` file sizes mismatch if I download from firefox

**Comment by Alevsk** [comment]
@dvaldivia @bexsoft this may be M1 related, I was able to generate, download and decompress the file on Mac OSX intel using safari

**Comment by Alevsk** [comment]
We should merge this, right now this feature is not working so users cannot generate the right file, If M1 users complain we can ask them to use Chrome/Firefox to download the file

**Review by dvaldivia — APPROVED** [review]
Tested.LGTM.

**Review by bexsoft — APPROVED** [review]
Approved as it seems like the issue is only with generated files on M1 CPU.

**Pull request merged**